helm/oauth2-proxy/values.yaml

```diff
@@ -62,6 +62,11 @@ image:
 # imagePullSecrets:
   # - name: myRegistryKeySecretName
 
+# Set a custom containerPort if required.
+# This will default to 4180 if this value is not set and the httpScheme set to http
+# This will default to 443 if this value is not set and the httpScheme set to https
+# containerPort: 4180
```

Comment:
- Could you also document the value in the README?
- Suggestion/discussion: Another simpler way of self-documenting would be to introduce a `containerPortHttp` and `containerPortHttps` with sane defaults always defined.

Reply:
@desaintmartin is the `README.md` enough for now?